docs/docs/import-external-csv-files-to-landing-zone.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Importing external CSV files to the Data Platform Landing Zone
+description: "Overview of how CSV files from external suppliers are imported to the Data Platform Landing Zone"
```

Comment:
This title is referencing CSV files however in the text it is only talking about liberator, which is a SQL dump.